android/beagle/src/test/java/br/com/zup/beagle/android/context/ContextDataEvaluationTest.kt

```diff
@@ -254,4 +249,58 @@ internal class ContextDataEvaluationTest {
         verify(exactly = once()) { BeagleMessageLogs.errorWhenExpressionEvaluateNullValue(any()) }
     }
 
-}
\ No newline at end of file
+    @Test
+    fun evaluateExpressionsForContext_should_set_binding_evaluatedExpressions_value_with_int_when_bind_type_is_int() {
+        //GIVEN
+        val bind = commonMock<Integer>()
+        every { bind.type } returns Integer::class.java
+
+        //WHEN
+        val result = contextDataEvaluation.evaluateBindExpression(
+            contextData = ContextData("context", 0.0),
+            bind = bind
+        )
+        //THEN
+        assertEquals(0, result)
```

Comment:
What do you think of validating the result type?
```suggestion
        assert(result is Integer)
```

Reply:
Nice suggestion, I'll make the change to the other tests too